modules/covid_research/spec/serializers/covid_research/genisis_serializer_spec.rb

```diff
@@ -24,8 +32,22 @@
       expect(output['UpdatedDateTime']).not_to be_empty
     end
 
+    it 'formats the times as iso8601' do
+      expected = Time.now.utc.iso8601
+
+      expect(output['CreatedDateTime']).to eq(expected.to_s)
+    end
+
     it 'translates the json payload to a list of key value pairs' do
       expect(output['FormQuestions']).to eq(expected)
     end
+
+    it 'translates true to "Yes"' do
+      expect(output['FormQuestions'].first['QuestionValue']).to eq 'No'
+    end
+
+    it 'translates false to "No"' do
+      expect(output['FormQuestions'][5]['QuestionValue']).to eq 'Yes'
+    end
```

Comment:
Should the Yes and No on the above tests be switched?

Reply:
Doh, good catch!  Just pushed up a fix for that.